apps/medusa-be/src/api/admin/promotions/README.md

```
@@ -53,13 +53,13 @@ Search and filter product variants for promotion rules.
 }
 ```
 
-### Producer Options
+### Brand Options
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)